webui/react/src/utils/experiment.ts

```diff
@@ -139,7 +139,11 @@ export const canExperimentContinueTrial = (
 ): boolean =>
   !experiment.archived &&
   !experiment.parentArchived &&
-  (!!trial || experiment?.numTrials === 1) &&
+  (!!trial ||
+    experiment?.numTrials === 1 ||
+    (experiment.state !== RunState.Completed &&
+      experiment.numTrials > 1 &&
+      ['random', 'grid'].includes(experiment.config?.searcher.name || ''))) &&
```

Comment:
maybe `['random', 'grid']` can be moved out to the component with a good variable name, then export it for both components?
and this can be set instead of array
i'll defer the decision to you

Reply:
Sure, added a new variable named `ContinuableNonSingleSearcherName`